frontend/package.json

```diff
@@ -70,8 +70,10 @@
     "lint:stylelint:write": "stylelint '**/*.scss' --fix",
     "test": "react-scripts test",
     "test:ci": "CI=true yarn test --coverage",
-    "e2e": "cd cypress && docker-compose up -d && docker-compose logs -f cypress && docker-compose down",
     "eject": "react-scripts eject",
+    "e2e": "./e2e.sh",
+    "e2e:open": "./e2e.sh -o true",
+    "e2e:compose": "cd cypress && sed \"s;\\(CHECK_COMMIT= *\\)\\(.*\\);\\1$(git rev-parse HEAD);\" .env.e2e > tmpfile && mv tmpfile .env.e2e && docker-compose up -d && docker-compose logs -f cypress && docker-compose down",
```

Comment:
Can you update [this line in the README](https://github.com/CDCgov/prime-simplereport/pull/3186/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R285) to point at the `yarn e2e:compose` command rather than `yarn:e2e`?